Source/WebKit/NetworkProcess/NetworkCORSPreflightChecker.cpp

```diff
@@ -103,7 +103,13 @@ void NetworkCORSPreflightChecker::didReceiveChallenge(WebCore::AuthenticationCha
     bool isTLSHandshake = scheme == ProtectionSpace::AuthenticationScheme::ServerTrustEvaluationRequested
         || scheme == ProtectionSpace::AuthenticationScheme::ClientCertificateRequested;
 
+#if USE(GLIB)
+    bool askUserForCredentials = scheme == ProtectionSpace::AuthenticationScheme::ClientCertificatePINRequested;
+
+    if (!askUserForCredentials && !isTLSHandshake) {
+#else
     if (!isTLSHandshake) {
+#endif
```

Comment:
I'm going to suggest writing this differently to avoid mixing `#if` with `if`:

```suggestion
#if USE(GLIB)
    bool askUserForCredentials = scheme == ProtectionSpace::AuthenticationScheme::ClientCertificatePINRequested;
#else
    bool askUserForCredentials = false;
#endif

    if (!askUserForCredentials && !isTLSHandshake) {
```

Reply:
Great suggestion, I like this a ton better too. Will add that now.